core/.js/src/test/scala/io/udash/routing/RoutingEngineTest.scala

```diff
@@ -2,106 +2,140 @@ package io.udash.routing
 
 import io.udash._
 import io.udash.testing._
+import org.scalactic.source.Position
 
 class RoutingEngineTest extends UdashFrontendTest with TestRouting {
 
   "RoutingEngine" should {
     "render valid views on url change" in {
```

Comment:
Could we split this test case? It's really long and description is very general.

Reply:
I considered it briefly, but it led to another test case of similar length - without a few assertions. I'll keep this as integration test for now.